README.md

```diff
@@ -59,6 +59,28 @@ once its done, push it to your image repository, or rename and push to a local r
 1. Create secret.txt with the exports from the section above (include the export command).
 
 2. Create a configmap for the txt file
```

Comment:
```suggestion
2. Create a configmap/secret for the txt file
```

Reply:
but this command is only for the configmap, secret has different progress below

Reply:
Yeah but the title "2. Create... " is for both no?

Reply:
yes and no, 2 as a whole talks about them both,
but the first line you quoted is for configmap only and refer to line 62 which is configmap only,
later in the section it tells how to use secret if desired